{
  "cryptography":[
      {
         "reason_to_ignore": "Temporarily ignoring to fix CRITICALs",
         "description":"Issue summary: Generating excessively long X9.42 DH keys or checking\nexcessively long X9.42 DH keys or parameters may be very slow.\n\nImpact summary: Applications that use the functions DH_generate_key() to\ngenerate an X9.42 DH key may experience long delays.  Likewise, applications\nthat use DH_check_pub_key(), DH_check_pub_key_ex() or EVP_PKEY_public_check()\nto check an X9.42 DH key or X9.42 DH parameters may experience long delays.\nWhere the key or parameters that are being checked have been obtained from\nan untrusted source this may lead to a Denial of Service.\n\nWhile DH_check() performs all the necessary checks (as of CVE-2023-3817),\nDH_check_pub_key() doesn't make any of these checks, and is therefore\nvulnerable for excessively large P and Q parameters.\n\nLikewise, while DH_generate_key() performs a check for an excessively large\nP, it doesn't check for an excessively large Q.\n\nAn application that calls DH_generate_key() or DH_check_pub_key() and\nsupplies a key or parameters obtained from an untrusted sour",
         "vulnerability_id":"CVE-2023-5678",
         "name":"CVE-2023-5678",
         "package_name":"cryptography",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/cryptography-41.0.5.dist-info/METADATA",
            "name":"cryptography",
            "package_manager":"PYTHONPKG",
            "version":"41.0.5",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-5678",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-5678 - cryptography"
      }
  ],
  "transformers": [
    {
      "reason_to_ignore": "Need guidance from HF team on how to upgrade as it's breaking the image. Temporarily ignoring to patch other CVEs.",
      "description": "\n Insecure Temporary File in GitHub repository huggingface/transformers prior to 4.30.0..",
      "vulnerability_id": "CVE-2023-2800",
      "name": "CVE-2023-2800",
      "package_name": "transformers",
      "package_details": {
        "file_path": null,
        "name": "transformers",
        "package_manager": "PYTHONPKG",
        "version": "4.26.0",
        "release": ""
      },
      "remediation": {
        "recommendation": {
          "text": "Update transformers to 4.30.0."
        }
      },
      "cvss_v3_score": 4.7,
      "cvss_v30_score": 4.7,
      "cvss_v31_score": 4.7,
      "cvss_v2_score": 4.7,
      "cvss_v3_severity": "MEDIUM",
      "source_url": "https://nvd.nist.gov/vuln/detail/CVE-2023-2800",
      "source": "NVD",
      "severity": "MEDIUM",
      "status": "ACTIVE",
      "title": "CVE-2023-2800 - transformers"
    }
  ],
  "libzmq5": [
    {
      "reason_to_ignore": "Version is passed the fix version from vendor",
      "description": "\n It was discovered that ZeroMQ incorrectly handled certain application metadata.\n A remote attacker could use this issue to cause ZeroMQ to crash, or possibly\n execute arbitrary code.",
      "vulnerability_id": "CVE-2019-13132",
      "name": "CVE-2019-13132",
      "package_name": "libzmq5",
      "package_details": {
        "file_path": null,
        "name": "libzmq5",
        "package_manager": "OS",
        "version": "4.3.2",
        "release": "2ubuntu1"
      },
      "remediation": {
        "recommendation": {
          "text": "None Provided"
        }
      },
      "cvss_v3_score": 9.8,
      "cvss_v30_score": 0,
      "cvss_v31_score": 9.8,
      "cvss_v2_score": 7.5,
      "cvss_v3_severity": "CRITICAL",
      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2019/CVE-2019-13132.html",
      "source": "UBUNTU_CVE",
      "severity": "HIGH",
      "status": "ACTIVE",
      "title": "CVE-2019-13132 - libzmq5"
    }
  ],
  "tensorflow-gpu":[
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, if the stride and window size are not positive for `tf.raw_ops.AvgPoolGrad`, it can give a floating point exception. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25669",
         "name":"CVE-2023-25669",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25669",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25669 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, when `ctx->step_containter()` is a null ptr, the Lookup function will be executed with a null pointer. A fix is included in TensorFlow 2.12.0 and 2.11.1.",
         "vulnerability_id":"CVE-2023-25663",
         "name":"CVE-2023-25663",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25663",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25663 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, there is a heap buffer overflow in TAvgPoolGrad. A fix is included in TensorFlow 2.12.0 and 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25664",
         "name":"CVE-2023-25664",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":9.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":9.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"CRITICAL",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25664",
         "source":"NVD",
         "severity":"CRITICAL",
         "status":"ACTIVE",
         "title":"CVE-2023-25664 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, integer overflow occurs when `2^31 <= num_frames * height * width * channels < 2^32`, for example Full HD screencast of at least 346 frames. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25667",
         "name":"CVE-2023-25667",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25667",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25667 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an end-to-end open source platform for machine learning. Constructing a tflite model with a paramater `filter_input_channel` of less than 1 gives a FPE. This issue has been patched in version 2.12. TensorFlow will also cherrypick the fix commit on TensorFlow 2.11.1.\n",
         "vulnerability_id":"CVE-2023-27579",
         "name":"CVE-2023-27579",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-27579",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-27579 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, if the parameter `indices` for `DynamicStitch` does not match the shape of the parameter `data`, it can trigger an stack OOB read. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25659",
         "name":"CVE-2023-25659",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25659",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25659 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Versions prior to 2.12.0 and 2.11.1 are vulnerable to integer overflow in EditDistance. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25662",
         "name":"CVE-2023-25662",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25662",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25662 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. The function `tf.raw_ops.LookupTableImportV2` cannot handle scalars in the `values` parameter and gives an NPE. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25672",
         "name":"CVE-2023-25672",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25672",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25672 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Versions prior to 2.12.0 and 2.11.1 have a null point error in QuantizedMatMulWithBiasAndDequantize with MKL enabled. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25670",
         "name":"CVE-2023-25670",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25670",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25670 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, when `SparseSparseMaximum` is given invalid sparse tensors as inputs, it can give a null pointer error. A fix is included in TensorFlow version 2.12 and version 2.11.1.",
         "vulnerability_id":"CVE-2023-25665",
         "name":"CVE-2023-25665",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25665",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25665 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Attackers using Tensorflow prior to 2.12.0 or 2.11.1 can access heap memory which is not in the control of user, leading to a crash or remote code execution. The fix will be included in TensorFlow version 2.12.0 and will also cherrypick this commit on TensorFlow version 2.11.1.",
         "vulnerability_id":"CVE-2023-25668",
         "name":"CVE-2023-25668",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":9.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":9.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"CRITICAL",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25668",
         "source":"NVD",
         "severity":"CRITICAL",
         "status":"ACTIVE",
         "title":"CVE-2023-25668 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source machine learning platform. Versions prior to 2.12.0 and 2.11.1 have a null pointer error in RandomShuffle with XLA enabled. A fix is included in TensorFlow 2.12.0 and 2.11.1.",
         "vulnerability_id":"CVE-2023-25674",
         "name":"CVE-2023-25674",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25674",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25674 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, an out of bounds read is in GRUBlockCellGrad. A fix is included in TensorFlow 2.12.0 and 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25658",
         "name":"CVE-2023-25658",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25658",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25658 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, when the parameter `summarize` of `tf.raw_ops.Print` is zero, the new method `SummarizeArray<bool>` will reference to a nullptr, leading to a seg fault. A fix is included in TensorFlow version 2.12 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25660",
         "name":"CVE-2023-25660",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25660",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25660 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source machine learning platform. Prior to versions 2.12.0 and 2.11.1, `nn_ops.fractional_avg_pool_v2` and `nn_ops.fractional_max_pool_v2` require the first and fourth elements of their parameter `pooling_ratio` to be equal to 1.0, as pooling on batch and channel dimensions is not supported. A fix is included in TensorFlow 2.12.0 and 2.11.1.",
         "vulnerability_id":"CVE-2023-25801",
         "name":"CVE-2023-25801",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.8,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.8,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25801",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25801 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Versions prior to 2.12.0 and 2.11.1 have a Floating Point Exception in TensorListSplit with XLA. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25673",
         "name":"CVE-2023-25673",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25673",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25673 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. There is out-of-bounds access due to mismatched integer type sizes. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.\n",
         "vulnerability_id":"CVE-2023-25671",
         "name":"CVE-2023-25671",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25671",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25671 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source machine learning platform. When running versions prior to 2.12.0 and 2.11.1 with XLA, `tf.raw_ops.Bincount` segfaults when given a parameter `weights` that is neither the same shape as parameter `arr` nor a length-0 tensor. A fix is included in TensorFlow 2.12.0 and 2.11.1.",
         "vulnerability_id":"CVE-2023-25675",
         "name":"CVE-2023-25675",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25675",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25675 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source platform for machine learning. Prior to versions 2.12.0 and 2.11.1, there is a floating point exception in AudioSpectrogram. A fix is included in TensorFlow version 2.12.0 and version 2.11.1.",
         "vulnerability_id":"CVE-2023-25666",
         "name":"CVE-2023-25666",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25666",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25666 - tensorflow-gpu"
      },
      {
         "description":"TensorFlow is an open source machine learning platform. When running versions prior to 2.12.0 and 2.11.1 with XLA, `tf.raw_ops.ParallelConcat` segfaults with a nullptr dereference when given a parameter `shape` with rank that is not greater than zero. A fix is available in TensorFlow 2.12.0 and 2.11.1.",
         "vulnerability_id":"CVE-2023-25676",
         "name":"CVE-2023-25676",
         "package_name":"tensorflow-gpu",
         "package_details":{
            "file_path":"opt/conda/lib/python3.9/site-packages/tensorflow_gpu-2.11.1.dist-info/METADATA",
            "name":"tensorflow-gpu",
            "package_manager":"PYTHONPKG",
            "version":"2.11.1",
            "release":null
         },
         "remediation":{
            "recommendation":{
               "text":"None Provided"
            }
         },
         "cvss_v3_score":7.5,
         "cvss_v30_score":0.0,
         "cvss_v31_score":7.5,
         "cvss_v2_score":0.0,
         "cvss_v3_severity":"HIGH",
         "source_url":"https://nvd.nist.gov/vuln/detail/CVE-2023-25676",
         "source":"NVD",
         "severity":"HIGH",
         "status":"ACTIVE",
         "title":"CVE-2023-25676 - tensorflow-gpu"
      }
  ]
}
